data-raw/config.R

```diff
@@ -1,7 +1,6 @@
-
 susceptible <- 1L
-infectious  <- 2L
-removed     <- 3L
+infectious <- 2L
+removed <- 3L
```

Comment:
i expect there's not a linter option for aligned declaration / assignment, but i have a more than just mild preference for that style.

Reply:
I'm not sure - probably involves a bit of a dig in the `lintr` docs.

Reply:
Sadly there is not and would also need to edit the style guide used by `styler`. Both of those things are doable but not trivially so.